proposals/nnnn-rpc-design-refactoring.md

```diff
@@ -0,0 +1,153 @@
+# RPC design refactoring
+
+* Proposal: [SDL-NNNN](nnnn-rpc-design-refactoring.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+This proposal is to rework design of application manager layer in order to:
+ - Reduce coupling between components
+ - Reduce responsibility of ApplicationManager class
```

Comment:
Reduce responsibility of ApplicationManager class
should become
Reduce responsibility of the ApplicationManager class

Reply:
@mrapitis, done in 2c9f7a3f7bb7e6c8b26bb1e4aec779cf65733666